modules/artifact-activation/src/test/resources/apps/lightweight/test-app-missing-gav-valid-parent/META-INF/maven/pom.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>test</groupId>
+    <artifactId>test-app-missing-gav-valid-parent</artifactId>
+    <version>1.0.0</version>
```

Comment:
This artifact has full GAV defined and does not have a parent. I think you need to use an artifact that is missing one of the coordinates and has a parent section defined so it takes them from there.
Otherwise the test will pass but you are not testing what you are supposed to be testing.